### DISCUSSION TITLE
Can CustomCSSforFx also be applied to Firefox Android [root]?

### DISCUSSION THREAD

**Discussion by LeeBinder** [Aris-t2/CustomCSSforFx#466] [Q&A] [answered]
And if so, where would the files need to be copied to? Would the Firefox (or Fennec, Iceraven etc.) apk itself need to be modified (and resigned), too? Or is that not possible in general at this project's current state?

Needless to say, prerequisite is a rooted system.

**Comment by Aris-t2** [comment, marked as answer]
Unfortunately the kind of customization offered by this project is for desktop systems only.

In theory it would be possible to unpack the apk file, the omni.ja file inside it an modify existing CSS files, but this would be an entirely different project.

**Comment by LeeBinder** [comment]
Thanks Aris for the clarification. Peeking into omni.ja, that indeed would be the target file

![Bildschirmfoto 2022-04-14 um 19 41 35](https://user-images.githubusercontent.com/39203497/163444157-af0f3deb-6b86-4138-98d2-817ead477977.jpg)

but I'll heed your hint that messing with this would most likely open a can of worms.

In Android, for those with an urge for customization, Firefox Nightly (amongst others) seems like one sensible Firefox alternative.
